Source/Private/ARTGCD.h

```diff
@@ -9,7 +9,13 @@
 #import <Foundation/Foundation.h>
 
 @interface ARTScheduledBlockHandle : NSObject
+- (instancetype)initWithDelay:(NSTimeInterval)delay queue:(dispatch_queue_t)queue block:(dispatch_block_t)block;
+- (void)cancel;
 @end
 
 ARTScheduledBlockHandle *artDispatchScheduled(NSTimeInterval seconds, dispatch_queue_t queue, dispatch_block_t block);
-void artDispatchCancel(ARTScheduledBlockHandle *handle);
+static inline void artDispatchCancel(ARTScheduledBlockHandle *handle) {
+    if (handle) {
```

Comment:
Just calling [handle cancel] should be fine. Check on NULL necessary on direct member access with -> operator. Just saying, can be ignored.

Reply:
True, but some (including myself) may prefer to avoid the "feature" of objective-C where a message to `nil` is _ignored_, and this is more explicit.

Reply:
PS: I started writing a bit of objective-C last week, so my opinion there is under developed 😅

Reply:
is this one could be resolved? @maratal